.gitignore

```diff
@@ -0,0 +1,2 @@
+
+style.css
```

Comment:
Why have you added `style.css` to your ignore file?

The `.gitignore` file is used to exclude certain files from being tracked in git. Since you'll be making changes to the `style.css` file, I think you want it to be tracked.

Instead, I think the files `.DS_Store` and `.vscode/settings.json` should be in your `.gitignore` file instead.

Reply:
I had no idea! I will fix it then Thank you

Reply:
Hi @Louise, Just wanted to let you know I am aware that I should include my css file on pull request as you explained but since I haven't added anything to it, I couldn't take it out of the ignore list I will do it in my next commit. Thank you :)